packages/theme-patternfly-org/helpers/configContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default React.createContext({});
```

Comment:
can you init it with context object keys set to null? so it's easier to tell what this context contains

Reply:
Yes, good idea!